server/repository/src/db_diesel/invoice_row.rs

```diff
@@ -106,7 +105,6 @@ pub struct InvoiceRow {
     pub clinician_id: Option<String>,
     pub currency_id: Option<String>,
```

Comment:
For what it's worth, Legacy mSupply seems to always populate this by default with the default currency. Always nice to remove an Option if unnecessary but maybe out of scope of this PR!